src/main/java/uk/gov/companieshouse/acsp/models/filing/ACSP.java

```diff
@@ -24,16 +24,12 @@ public final class ACSP {
     private String acspType;
     @JsonProperty("business_sector")
     private String businessSector;
-    @JsonProperty("amlmemberships")
-    private AmlMembership[] amlMemberships;
-    @JsonProperty("person_name")
-    private PersonName personName;
+    @JsonProperty("aml")
+    private Aml aml;
     @JsonProperty("business_name")
     private String businessName;
```

Comment:
as we have "business_name" which corresponds to proposedCorporateBodyName tag in xml, I not sure what is the purpose of the field company_name ? sorry, if I am missing something

Reply:
We get company_name in companyDetails object but business_name is basically used when we ask users if their business is registered with their name or some other name. Hence 2 different fields in front end.

Reply:
Yes, but does filling endpoint need this field? as it need to return data required for chips-filing-consumer. And we are not mapping company_name anywhere in json to xml conversion

Reply:
Also we are populating company_name and business_name in different ways and I would like to keep it separate anyway just so if we have to make changes to one then other doesn't get affected.